app/utils/index.ts

```diff
@@ -1,21 +1,28 @@
-class Generator {
+class SequentialIntegerGenerator {
+  private START: number = 0;
```

Comment:
```suggestion
  private static START: number = 0;
```

That's when it becomes a class prop, no?

Reply:
This would be fine, but I didn't want to write long lines `SequentialIntegerGenerator.START`. Maybe I would destructure.